README.md

```diff
@@ -108,6 +127,9 @@ command invocation).  If you require stronger protection against being called by
 decommissioned/deregistered service nodes from a connection established when the SN was active then
 the callback itself will need to verify when invoked.
 
+[//]: # decommissioned/deregistered SNs: sounds like an important use case, maybe we should include
+[//]: # this as design goal
```

Comment:
In practice this isn't really a big deal: the actual commands are going to need to verify whatever is being sent to them anyway.  For example, with blink messages we verify signatures included in the messages.  This is mainly just an optimization to not allow random clients to hit service node endpoints at all.